CONTRIBUTING.md

```diff
@@ -0,0 +1,100 @@
+- Push the changes to your remote fork.
+- Create a [pull request (PR)](https://github.com/cbrnr/sleepecg/pulls).
+- Add an entry to `CHANGELOG.md` (section 'UNRELEASED') where you mention the corresponding PR and (if you desire) your name.
+
+
+### Rebasing
+If another PR is merged while you are working on something, a merge conflict may arise. To resolve it, perform the following steps in your local clone:
+- Switch to the main branch: `git switch main`.
+- Pull the current changes: `git pull upstream main`.
+- Switch back to the branch you are working on: `git switch branch-name`.
+- Rebase your commits onto main: `git rebase main`.
+- Push to your remote fork: `git push` (might require `git push --force`).
+
+
+## Development enviroment
+Make sure to use Python 3.7. You might want to [create a virtual environment](https://docs.python.org/3/library/venv.html#creating-virtual-environments) instead of installing everything into your main environment. In the root of the local clone of your fork, run
+
+```
+pip install -e .[dev]
+```
+
+When using the flag [`-e`](https://pip.pypa.io/en/stable/cli/pip_install/#install-editable), pip does not copy the package is to `site-packges`, but creates a link to your local repository. So any changes to the sourcecode are directly reflected in the 'installed' package. Installing the optional `[dev]` dependencies makes sure all tools for style checking, testing and building documentation locally are available.
+
+
+## Code style
+SleepECG adheres to [PEP 8](https://www.python.org/dev/peps/pep-0008/), with the following exceptions/specifications:
+- Each source file contains a header listing the authors contributing to that module and mentions the license:
+    ```python
+    # Authors: Firstname Lastname
+    #          Another Name
+    #
+    # License: BSD (3-clause)
+    ```
+- The maximum line length is `92` (instead of `79`).
+    - If readability would suffer from a linebreak, append `# noqa` to the relevant line to disable line length checking.
+- Single quotes are used, except if the string contains an apostrophe.
+- [Type hints](https://www.python.org/dev/peps/pep-0484/) are encouraged.
+- If a container literal, function definition or function call does not fit on one line:
+    - Each argument or item is indented one level further than the function or container name.
+    - The last argument or item has a trailing comma
+    - The closing parenthesis or bracket is indented at the same level as the starting line.
+    ```python
+    # Example
+    def download_file(
+        url: str,
+        target_filepath: Path,
+        checksum: Optional[str] = None,
+        checksum_type: Optional[str] = None,
+        verbose: bool = False,
+    ) -> None:
+    ```
+
+
+## Documentation
+For docstrings, SleepECG mainly follows [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html), with the following exceptions/specifications:
+- The maximum line length is `75`. You can use
```

Comment:
Did you want to add anything here?

Reply:
I thought about recommending [rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap), but decided against it since it's specific to VSCode.
```suggestion
- The maximum line length is `75`.
```